### ISSUE TITLE
REQ: Option - Show active guide on indented column rather than the parent/previous

### ISSUE THREAD

**Issue by codeitco** [harai/indent-guide-improved#53]
Hi @harai,

Just wondering if it would be possible to add the option for allowing the active guide to to be highlighted on the indent column where the cursor is rather than always showing on parent/previous column of the cursor spot? This could be in addition to the current implementation of course, an option for either or.

Sometimes I like to click a long open element, and follow its entire guide down to its close tag and that is where my request would come in handy as the eyes could easily see the entire block of code easier from open-to-close.

Hope you will consider.

Thanks!

**Comment by harai** [comment]
Is this request same as #49 ?

**Comment by scsmash3r** [comment]
@harai Nope. I think here is what the author means, and that's the same issue for me: 
![indent_explain](https://cloud.githubusercontent.com/assets/4781532/21129459/47924d34-c10a-11e6-909a-8c31fed90919.jpg)

**Comment by codeitco** [comment]
Sorry for my lack of response/confirmation. scsmash3r's screenshot explains exactly what I was trying to say.

Thanks for the assist. ☺

**Comment by DariyGRAY** [comment]
...and that's the same issue for me!

**Comment by chenasraf** [comment]
+1, is this being planned? Or can anyone point me in the right direction to maybe work on this?

**Comment by ikushlianski** [comment]
+1, this feature will definitely make it closer to Notepad++'s functionality

**Comment by bgarchow** [comment]
Just installed the package and this issue is the first thing I noticed. The highlighted (active) guide should be the guide at the indent level the cursor is on. It really doesn't make much sense otherwise.

**Comment by GitHyp** [comment]
Shame this never got resolved... without the proper indent highlight it makes little sense to use this. The only thing N++ and Sublime do better than Atom. :(
